src/meta/IndexManager.cpp

```diff
@@ -0,0 +1,59 @@
+/* Copyright (c) 2019 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+#include "meta/IndexManager.h"
+
+namespace nebula {
+namespace meta {
+
+std::unique_ptr<IndexManager> IndexManager::create() {
+    return std::make_unique<IndexManager>();
+}
+
+void IndexManager::init(MetaClient *client) {
+    CHECK_NOTNULL(client);
+    metaClient_ = client;
+}
+
+StatusOr<std::shared_ptr<const cpp2::IndexFields>>
```

Comment:
For` cpp2::IndexFields`，Do we really need to use map at the moment?
struct IndexFields {
    1: map<string, list<common.ColumnDef>>(cpp.template = "std::map")  fields,
}